src/index.js

```diff
@@ -38,26 +44,18 @@ export default function () {
 
         reportTestDone (name, testRunInfo) {
             var hasErr = !!testRunInfo.errs.length;
-
-            if (testRunInfo.unstable)
-                name += ' (unstable)';
-
-            if (testRunInfo.screenshotPath)
-                name += ` (screenshots: ${testRunInfo.screenshotPath})`;
-
-            name = this.escapeHtml(name);
-
+            
             var openTag = `<testcase classname="${this.currentFixtureName}" ` +
-                          `name="${name}" time="${testRunInfo.durationMs / 1000}">\n`;
+                          `name="${this.escapeHtml(name)}" time="${testRunInfo.durationMs / 1000}">\n`;
 
             this.report += this.indentString(openTag, 2);
 
             if (testRunInfo.skipped) {
                 this.skipped++;
                 this.report += this.indentString('<skipped/>\n', 4);
             }
-            else if (hasErr)
-                this._renderErrors(testRunInfo.errs);
+            else if (hasErr || testRunInfo.unstable)
```

Comment:
You shouldn't mark all unstable tests as failed. It just makes quarantine mode useless. Please note, that if an unstable test is considered passed by TestCafe, reporter won't get any error from failed attempts.

Reply:
Yes this was a conflict that I had without appending it to the test name if the test is unstable. I'm not exactly sure of how quarantine mode works to be honest.

jUnit spec allows a testcase to have a status attribute that the 'unstable' status could go in. Would this work?
````
        <testcase assertions="" classname="" name="" status="" time="">
            <skipped/>
            <error message="" type=""/>
            <failure message="" type=""/>
            <system-out/>
            <system-err/>
        </testcase>
````

Reply:
Judging by the _de-facto_ official [JUnit schema](https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd) there is no `status` attribute in the `testcafe` element. Looks like it is a JUnit5 extension, judging by the [JUnit5 schema](https://github.com/junit-team/junit5/blob/master/platform-tests/src/test/resources/jenkins-junit.xsd). Maybe we can store the `unstable` flag in the `<system-out>` or the `<system-err>`.

Reply:
Yes, in that repo from windyroad, there is no `status` attribute. There seems to be some inconsistency out there with the schema, as the schema for [junit4](https://github.com/behave/behave/blob/master/etc/junit.xml/junit-4.xsd) (released in 2006) also has the `status` attribute.
I'm not sure what the de-facto is and where it is used as there is a bit of variation. But I agree the  `<system-out>` may be more compatible.